launcher/src/main/java/com/skcraft/launcher/model/minecraft/Arguments.java

```diff
@@ -0,0 +1,16 @@
+package com.skcraft.launcher.model.minecraft;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * @author barpec12
+ */
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Arguments {
+
+    private List<Object> game;
```

Comment:
https://github.com/SKCraft/Launcher/pull/270/#issuecomment-449992857
```suggestion
     private List<Object> game;
     private List<Object> jvm;
```